client/src/pages/Balance.js

```diff
@@ -1,4 +1,4 @@
-import React, { useContext, useState } from "react";
+import React, { useEffect, useContext, useState, useReducer } from "react";
```

Comment:
since this is not using the global loading state, should add a local loading state instead